audio/descriptors.c

```diff
@@ -26,10 +28,13 @@ __code __at(DSCR_AREA) struct usb_descriptors code_descriptors = {
         .bLength            = USB_DT_DEVICE_SIZE,
         .bDescriptorType    = USB_DT_DEVICE,
         .bcdUSB             = USB_BCD_V20,
-        .bDeviceClass       = 0, // Class and protocol defined per interface
+        /* Class defined per interface */
```

Comment:
Why is the device/subclass zero? I think you want "Unused, the class is instead defined in the interface" or something similar...

Reply:
0 indicates the class is defined per interface

Reply:
See, I had forgotten that and the comment didn't tell me that...

Reply:
Fixed